samples/app/src/androidTest/java/com/sourcepoint/app/v6/MainActivityKotlinNativeMessage.kt

```diff
@@ -37,32 +32,11 @@ import org.koin.core.context.loadKoinModules
 class MainActivityNativeMessTest {
 
     lateinit var scenario: ActivityScenario<NativeMessageActivity>
-
-    @MockK
-    lateinit var spClient: SpClient
```

Comment:
just a general question, because I know that this PR is basically a big _REVERT_ to the old state:
is there any advantage in initializing mocks in each and every test case, instead of initializing it like the way we do right now?

Reply:
Yes, mock objects retain their state, this means that the checks after the first usage give you the wrong outcome